### ISSUE TITLE
Move to fork? Yes/no - Result is Yes, Alternate fork in comments

### ISSUE THREAD

**Issue by AlUlkesh** [yfszzx/stable-diffusion-webui-images-browser#82]
Well, this repository's owner seems to be on an extended hiatus. Hope he's okay.

Should we move to a different fork for now, so we can solve the accumulated issues?

https://poll-maker.com/poll4678984x1a434787-146

**Comment by toyxyz** [comment]
About a month has passed since there was no response from yfszzx. Maybe it's just that he's so busy with work that he doesn't have time to care. I think it would be better to move to another fork.

**Comment by AlUlkesh** [comment]
Well, everyone who voted, voted with Yes.

I have setup this fork, that has all open pull requests already merged in:

https://github.com/AlUlkesh/stable-diffusion-webui-images-browser

**Comment by ModischFabrications** [comment]
You might want to request a URL change on the automatic1111 repo, I just stumbled upon the branch after installing from the UI and getting weird errors.

**Comment by AlUlkesh** [comment]
Yes, the maintainer of another major fork and I are currently trying to merge those two. When that's done, one of the two will go into the webui's extension list.

**Comment by Testertime** [comment]
> Well, everyone who voted, voted with Yes.
> 
> I have setup this fork, that has all open pull requests already merged in:
> 
> https://github.com/AlUlkesh/stable-diffusion-webui-images-browser

I'm late to the party, but thank you for doing so! The Automatic1111 web UI development is rather fast, with occassional breaking changes. So this is a wise decision 👍
